openid-connect-server/src/test/java/org/mitre/oauth2/service/impl/TestDefaultIntrospectionResultAssembler.java

```diff
@@ -255,8 +270,8 @@ public void shouldAssembleExpectedResultForRefreshTokenWithoutUserInfo() throws
 		assertThat(result, is(equalTo(expected)));
 	}
 
-	@Test
-	public void shouldAssembleExpectedResultForRefreshTokenWithoutExpiry() {
+	@Test(expected = ParseException.class)
+	public void shouldAssembleExpectedResultForRefreshTokenWithoutExpiry() Throwable{
```

Comment:
Same comment as above, this test should not throw an exception. To test the code that will throw an exception, create a new `@Test`.